crates/fuzzing/wasm-spec-interpreter/README.md

```diff
@@ -0,0 +1,44 @@
+wasm-spec-interpreter
+=====================
+
+This project shows how to use `ocaml-interop` to call into the Wasm spec interpreter. There are
+several steps to making this work:
+ - building the OCaml Wasm spec interpreter as a static library
+ - building a Rust-to-OCaml FFI bridge using `ocaml-interop` and a custom OCaml wrapper
+ - linking both things into a Rust crate
+
+### Dependencies
+
+This crate only builds in an environment with:
+- `make` (the Wasm spec interpreter uses a `Makefile`)
+- `ocamlopt`, `ocamlbuild` (available with, e.g., `dnf install ocaml`)
+- Linux tools like `cp` and `ar`; currently it is easiest to build the static libraries in a single
+  environment but this could be fixed in the future (TODO)
```

Comment:
This crate has some dependencies that not all users will have: how do we want to configure this so that it will build and run by default when it can but not fail otherwise?

Reply:
I haven't dug through the Cargo-related details yet but IIUC, this is only built when invoking the particular fuzz target with `cargo fuzz run differential_spec`, right? If so, I think it's fine -- someone is unlikely to stumble on it by accident.

Reply:
Hm, well, I added `--exclude wasm-spec-interpreter` in a few places but I'm still getting errors. I think the problem is that `wasmtime-fuzzing` is being built and tested in various places and it pulls in `wasm-spec-interpreter` as a dependency.